src/app.tsx

```diff
@@ -1,6 +1,6 @@
 // lib imports
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import React from 'react';
```

Comment:
import statements should be changed back to namespace, as we want to import namespace and not default export